vassal-app/src/main/java/VASSAL/build/module/properties/PropertyChangerConfigurer.java

```diff
@@ -101,7 +101,7 @@ public Component getControls() {
         updateControls();
       };
       controls = new JPanel();
-      controls.setLayout(new MigLayout("ins 0,hidemode 3", "[]rel[][]rel[fill,grow][fill,grow]")); // NON-NLS
+      controls.setLayout(new MigLayout("debug,ins 0,hidemode 3", "[]rel[][]rel[fill,grow][fill,grow]")); // NON-NLS
```

Comment:
Did you leave that `debug` there accidentally?

Reply:
Yes indeed! Fixed.